### ISSUE TITLE
Database 'KNOEMA_ECONOMY_DATA_ATLAS' does not exist or not authorized

### ISSUE THREAD

**Issue by ramaKrishna0688** [Snowflake-Labs/sfquickstarts#278]
I am new to snowflake and trying to learn it by looking at snow exam guide for data engineer. Start practicing snowflake using KNOEMA_ECONOMY_DATA_ATLAS database using DBT.
But while creating my first pipe getting below error. Followed all the steps mentioned in the documentation.

Database 'KNOEMA_ECONOMY_DATA_ATLAS' does not exist or not authorized

Could any one guide me to resolve it. 

Please let me know if any additional info required.

**Comment by NickeyKim** [comment]
got same issue, the lab pages of Snowflake are no longer supported now?

**Comment by yarodmi** [comment]
Good catch! We are updating the guide, the mounted database is going to be called ECONOMY_DATA_ATLAS by default, once you mount it from the marketplace.

**Comment by mtulow** [comment]
I am still running into the same issue of showing 'does not exist or not authorized.' as an error message. Was anybody able to fix the bug?
